l10n_th_tin_data/readme/USAGE.rst

```diff
@@ -0,0 +1,75 @@
+***********
+Preparation
```

Comment:
This section, I would say it is "Configuration", should go to CONFIGURE.rst file.

Reply:
From here http://www.rd.go.th/publish/42533.0.html
At the bottom, download section.  I need to combine 2 certs into 1.
But depends on SSL version (or related python library), requests.session may have an error. That is why the in the exception it may fall back to session.verify = False (not use cert).